### DISCUSSION TITLE
2 different Wyze Cam v3 boot up failures

### DISCUSSION THREAD

**Discussion by jrgilman** [gtxaspec/wz_mini_hacks#511] [Q&A]
I have had a single Wyze Cam v3 that's more than a year old (4.36.9.139) running wz_mini for a while now on a 16GB no-name MicroCenter SD card.  I've been wanting to expand my cameras so I grabbed two more the other day from Home Depot. I also grabbed two 32GB SanDisk cards which I tested using the f3 utility.

One camera came with 4.36.9.139 but the other one came with 4.36.10.X which I manually downgraded to 4.36.9.139.

Both cameras work fine when set-up with the Wyze app, but things go side ways fast when wz_mini_hacks gets loaded. I used the exact configuration from my older camera (literally copy and pasted the file from the old sd card) and only changed the hostname:

```
➜  Desktop cat wz_mini.conf 
#wz_mini configuration


#### W E B CAMERA###
##THIS MODE DISABLES EVERYTHING AND IT WILL
## WORK AS A WEB CAMERA FOR YOUR PC ***ONLY***
WEB_CAM_ENABLE="false"
WEB_CAM_BIT_RATE="8000"
WEB_CAM_FPS_RATE="25"

#####NETWORKING#####
CUSTOM_HOSTNAME="WCV3-SUNROOM"

ENABLE_USB_ETH="false"
ENABLE_USB_ETH_MODULE_AUTODETECT="true"
ENABLE_USB_ETH_MODULE_MANUAL=""

ENABLE_USB_DIRECT="false"
USB_DIRECT_MAC_ADDR="02:01:02:03:04:08"

ENABLE_USB_RNDIS="false"

ENABLE_IPV6="false"

ENABLE_WIREGUARD="false"
WIREGUARD_IPV4=""
WIREGUARD_PEER_ENDPOINT=""
WIREGUARD_PEER_PUBLIC_KEY=""
WIREGUARD_PEER_ALLOWED_IPS=""
WIREGUARD_PEER_KEEP_ALIVE=""

ENABLE_IPTABLES="false"

ENABLE_NFSv4="false"

ENABLE_RTL8189FS_DRIVER="true"
ENABLE_ATBM603X_DRIVER="true"

NTP_SERVER=

##### NETWORK INTERFACE BONDING #####
BONDING_ENABLED="false"
BONDING_PRIMARY_INTERFACE="eth0"
BONDING_SECONDARY_INTERFACE="wlan0"
BONDING_LINK_MONITORING_FREQ_MS="100"
BONDING_DOWN_DELAY_MS="5000"
BONDING_UP_DELAY_MS="5000"
BONDING_FAIL_OVER_MAC="0"

#####ACCESSORIES#####
REMOTE_SPOTLIGHT="false"
REMOTE_SPOTLIGHT_HOST="0.0.0.0"

#####VIDEO STREAM#####
RTSP_LOGIN="admin"
RTSP_PASSWORD="<password>"
RTSP_PORT="8554"

RTSP_HI_RES_ENABLED="true"
RTSP_HI_RES_ENABLE_AUDIO="false"
RTSP_HI_RES_FPS=""
RTSP_HI_RES_MAX_BITRATE=""
RTSP_HI_RES_TARGET_BITRATE=""
RTSP_HI_RES_ENC_PARAMETER=""

RTSP_LOW_RES_ENABLED="true"
RTSP_LOW_RES_ENABLE_AUDIO="false"
RTSP_LOW_RES_FPS=""
RTSP_LOW_RES_MAX_BITRATE=""
RTSP_LOW_RES_TARGET_BITRATE=""
RTSP_LOW_RES_ENC_PARAMETER=""

RTSP_AUTH_DISABLE="false"

#####RTMP STREAMING#####
#### REQUIRES RTSP ####
### AND RTSP AUDIO ###
RTMP_STREAM_ENABLED="false"
RTMP_STREAM_FEED="video1_unicast"
RTMP_STREAM_SERVICE="youtube"
RTMP_STREAM_DISABLE_AUDIO="false"
RTMP_STREAM_YOUTUBE_KEY=""
RTMP_STREAM_TWITCH_KEY=""
RTMP_STREAM_FACEBOOK_KEY=""

#####GENERAL#####
LIBCALLBACK_ENABLE="true"
ENABLE_SWAP="true"
ENABLE_USB_STORAGE="false"
ENABLE_EXT4="false"
ENABLE_CIFS="false"
DISABLE_FW_UPGRADE="true"
AUDIO_PROMPT_VOLUME="50"
ENABLE_MP4_WRITE="false"
NIGHT_DROP_DISABLE="false"
DISABLE_MOTOR="false"
ENABLE_FSCK_ON_BOOT="false"
ENABLE_CAR_DRIVER="false"
ENABLE_LOCAL_DNS="false"
ENABLE_CRONTAB="false"
ENABLE_SELFHOSTED_MODE="false"

#####DEBUG#####
#drops you to a shell via serial, doesn't load app_init.sh
DEBUG_ENABLED="false"
DEBUG_INITRAMFS_ENABLED="false"
DEBUG_PASSWORD="false"
ENABLE_SYSLOG_SAVE="false"

####WEB####
WEB_SERVER_ENABLED="false"

#####SCRIPTING#####
CUSTOM_SCRIPT_PATH=""
```

Both cameras seem to fail to boot in two different ways.

For the downgraded camera, I hear the initial WZ Mini first boot prompt along with the swap prompt but it just sits there afterwards with a red LED (which my working camera also does) but no SSH access or web server (I've tried to modify the config to enable that).

The camera which came with the right firmware out of the box, I hear the same initial prompt as the other camera but it eventually settles on an alternating red/blue LED. Based on the Wyze docs this means it's unable to find a connection - although when I pull the SD card it boots up just fine and connects within seconds. Trying to connect to it is also a no-go.

I've tried turning on serial debugging and plugging it into my USB port but it seems to draw a bit too much power from my laptop's USB port to properly boot so I need to wait on an injector.

Any help is greatly appreciated.

**Comment by Bob-AOL** [comment]
My money is on the SD cards and their formatting. Copy and paste can also be problematic.

That said, if the copy/paste was done in a Windows environment, you may want to check the formatting of the wz_mini.conf file itself for Unix (LF) or Windows (CRLF) conventions. Gotta be Unix (LF).

I'd start from scratch right from the distro for each SD card individually and see where that gets you.

It's something simple.
